dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.sessionCache1.0.jakarta.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = io.openliberty.sessionCache1.0.jakarta
+visibility = private
+
+-bundles= com.ibm.ws.session.jakarta, \
+  		  com.ibm.ws.session.cache.jakarta, \
+  		  com.ibm.ws.session.store.jakarta, \
+  		  com.ibm.websphere.javaee.jcache.1.1.jakarta; mavenCoordinates="javax.cache:cache-api:1.1.0"
```

Comment:
What is the deal with the maven coordinates.  Seems wrong to use javax here.

Reply:
This bundle for there in the original public feature. And javax.cache doesn't come under jakartaee 9.